pages/common/you-get.md

```diff
@@ -0,0 +1,24 @@
+# you-get
+
+> Download media contents (videos, audios, images) from the Web.
+> More information: <https://you-get.org>.
+
+- Print media information about a specific media on the web:
+
+`you-get --info {{https://example.com/video?id=value}}`
+
+- Download a media from a specific URL:
+
+`you-get {{https://example.com/video?id=value}}`
+
+- Search on Google Videos and download:
+
+`you-get {{keywords}}`
+
+- Download a media to a specific location:
+
+`you-get --output-dir {{path/to/directory}} --output-filename {{path/to/filename}} {{https://example.com/watch?v=value}}`
```

Comment:
The container folder is defined by `--output-dir`, the `--output-filename` is used only for the `filename` itself.
```suggestion
`you-get --output-dir {{path/to/directory}} --output-filename {{filename}} {{https://example.com/watch?v=value}}`
```

Reply:
I'll apply this little fix, and merge this.